src/js/msp.js

```diff
@@ -193,8 +288,40 @@ var MSP = {
             bufView[4] = code; // code
             bufView[5] = bufView[3] ^ bufView[4]; // checksum
         }
+        return bufferOut;
+    },
+    encode_message_v2: function (code, data) {
+        const dataLength = data ? data.length : 0;
+        // 9 bytes for protocol overhead
+        const bufferSize = dataLength + 9;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'const' statement.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5635143)